airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/UsagePerConnectionTable.tsx

```diff
@@ -105,6 +114,152 @@ const UsagePerConnectionTable: React.FC<UsagePerConnectionTableProps> = ({ credi
+                  {props.cell.getValue()}
+                </Text>
+              </NavLink>
+            ),
+          }),
+          columnHelper.accessor("sourceConnectionName", {
+            header: () => (
+              <SortableTableHeader
+                onClick={() => onSortClick("sourceConnectionName")}
+                isActive={sortBy === "sourceConnectionName"}
+                isAscending={sortOrder === SortOrderEnum.ASC}
+              >
+                <FormattedMessage id="credits.source" />
+              </SortableTableHeader>
+            ),
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+            cell: (props) => (
+              <NavLink
+                to={`/${RoutePaths.Workspaces}/${workspaceId}/${RoutePaths.Source}/${props.row.original.sourceId}`}
+              >
+                <FlexContainer direction="row" alignItems="center">
+                  <ConnectorIcon icon={props.row.original.sourceIcon} />
+                  <Text size="sm" className={styles.cellText}>
+                    {props.cell.getValue()}
+                  </Text>{" "}
+                </FlexContainer>
+              </NavLink>
+            ),
+          }),
+          columnHelper.display({
+            id: "arrow",
+            cell: () => <ArrowRightIcon />,
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+          }),
+          columnHelper.accessor("destinationConnectionName", {
+            header: () => (
+              <SortableTableHeader
+                onClick={() => onSortClick("destinationConnectionName")}
+                isActive={sortBy === "destinationConnectionName"}
+                isAscending={sortOrder === SortOrderEnum.ASC}
+              >
+                <FormattedMessage id="credits.destination" />
+              </SortableTableHeader>
+            ),
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+            cell: (props) => (
+              <NavLink
+                to={`/${RoutePaths.Workspaces}/${workspaceId}/${RoutePaths.Destination}/${props.row.original.destinationId}`}
+              >
+                <FlexContainer direction="row" alignItems="center">
+                  <ConnectorIcon icon={props.row.original.destinationIcon} />
+                  <Text size="sm">{props.cell.getValue()}</Text>
+                </FlexContainer>{" "}
+              </NavLink>
+            ),
+          }),
+          columnHelper.display({
+            id: "schedule",
+            header: () => (
+              <SortableTableHeader
+                onClick={() => {
+                  return null;
+                }}
+                isActive={false}
+                isAscending={false}
+              >
```

Comment:
Will schedule actually be sortable? Probably not super easy with cron expressions. If not, could we just make this a non-interactive table header?

Reply:
Good call.  I was going off the connections list table as a reference for this column and it looks non-interactive there as well.